modules/graphdb/user_data.tf

```diff
@@ -118,4 +118,13 @@ data "cloudinit_config" "entrypoint" {
       app_configuration_endpoint : var.app_configuration_endpoint
     })
   }
+
+  # 09 Execute additional scripts
+  dynamic "part" {
```

Comment:
I wonder if these should be before starting GraphDB. For example, if one of these downloads a JDBC driver for GraphDB, it should be done before starting the database. Otherwise you'd have to restart it a second time in the custom scripts?

Reply:
I would rather not, we have no idea how clients would use those and might cause problems.